mod/cp_notifications/start.php

```diff
@@ -673,9 +678,14 @@ function cp_create_annotation_notification($event, $type, $object) {
 					'cp_comment_from' => "<a href='{$content->getURL()}'>{$content_title}</a>",
 					'cp_msg_type' => 'cp_likes_topic_replies',
 				);
-				$author = get_entity($comment_author->getGUID());
-				$content_entity = $content;
-				$to_recipients[$comment_author->getGUID()] = $comment_author;
+				$author = $liked_by;
+				$content_entity = $content_entity;
+				$action_type = "like_reply";
+
+	    		if (strcmp(elgg_get_plugin_user_setting('cpn_likes_email', $comment_author->getGUID(),'cp_notifications'),'likes_email') == 0)
```

Comment:
Probably safer to use === for these comparisons, just in case the setting somehow becomes an array for example.